cartographer_ros/src/cartographer_node_main.cc

```diff
@@ -97,12 +98,91 @@ constexpr char kImuTopic[] = "/imu";
 constexpr char kOdometryTopic[] = "/odom";
 constexpr char kOccupancyGridTopic[] = "/map";
 
+struct NodeOptions {
+  carto::mapping::proto::MapBuilderOptions map_builder_options;
+  string tracking_frame;
+  string odom_frame;
+  string map_frame;
```

Comment:
Move this two lines up to be consistent with the new config and the CreateNodeOptions() code?